contracts/protocore/Protocore.sol

```diff
@@ -136,6 +163,10 @@ contract Protocore is MosaicVersion, ValidatorSet, CoconsensusModule {
         openKernelHeight = openKernelHeight.add(1);
         openKernelHash = _kernelHash;
 
+        Quorum storage quorum = quorums[openKernelHeight];
```

Comment:
the quorum can be tracked for each ForwardValidatorSet per metablock height; then there s no double storage or calculation.